src/rpc/error.rs

```diff
@@ -44,6 +45,7 @@ impl Errors {
             Errors::InvalidOrBlockedHost => {
                 (110, "Invalid or blocked destination host".to_string())
             }
+            Errors::TimedOut => (111, "Connection/Request/Response from the destination timedout".to_string())
```

Comment:
```suggestion
            Errors::TimedOut => (111, "Connection/Request/Response from the destination timed out".to_string())
```

Reply:
thanks I was a bit confused about this I should have just clicked commit suggestion before pushing the change from my local  but i applied your suggestion to my commit